spring-integration-core/src/test/java/org/springframework/integration/transformer/ObjectToMapTransformerTests.java

```diff
@@ -283,23 +339,21 @@ public void setDepartments(List<String> departments) {
 		private List<Map<String, Object>> remarks;
 		private Child child;
 		private BigDecimal age;
+		private Date birthDate;
+		public Instant deathDate;
+		private Address address;
 		public BigDecimal getAge() {
 			return age;
 		}
-
```

Comment:
Please, be sure to keep blank lines between method definitions.
I also prefer to have them as well in between field declarations.
The properties should be accessed via `this.` prefix.

Not critical and can be polished during merge, but for letting you to know what code style we pursue.